charts/cloudnative-pg/values.yaml

```diff
@@ -29,6 +29,9 @@ imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
 
+hostNetwork: false
+dnsPolicy: "ClusterFirst"
```

Comment:
How is `ClusterFirst` better than `Default` with `hostNetowork: false`?

Reply:
Shouldn't it be kept to Default for backwards compatibility?

Reply:
I think we could simply keep it as an empty string which will result in the field not being rendered in the Deployment.

However, from https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy, it seems like `ClusterFirst` is the actual default if the field is not specified in the PodSpec.

Reply:
Changed the default value of `dnsPolicy` to an empty string: https://github.com/cloudnative-pg/charts/pull/324/commits/825684865ef45bc47a92169673796b45e4a92a38